apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLoggerFactoryExtensions.cs

```diff
@@ -50,13 +51,15 @@ public static class GoogleLoggerFactoryExtensions
         ///     detected from the platform.</param>
         /// <param name="options">Optional, options for the logger.</param>
         /// <param name="client">Optional, logging client.</param>
+        /// <param name="accessor">Optional, HTTP context accessor. Allows adding trace ids to log entries.</param>
```

Comment:
Adding an extra parameter is a breaking change. Unfortunately adding an overload is tricky too - let's talk about this on a VC or similar.